internal/configs/version2/nginx.transportserver.tmpl

```diff
@@ -5,7 +5,7 @@ upstream {{ $u.Name }} {
     random two least_conn;
 
     {{ range $s := $u.Servers }}
-    server {{ $s.Address }} max_fails={{ $s.MaxFails }} fail_timeout={{ $s.FailTimeout }};
+    server {{ $s.Address }} max_fails={{ $s.MaxFails }} fail_timeout={{ $s.FailTimeout }} max_conns=$s.MaxConnections;
```

Comment:
there is a a bug here -- this will generate `max_conns=$s.MaxConnections;`

additionally, in this the Plus template, `if` is used, but here it doesn't use. Could we have a consistent approach?

`max_conns={{ $s.MaxConnections }};` should work without if, since the default is `0`.

Reply:
ah yes, will update.